### DISCUSSION TITLE
Release preview versions

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#1337] [Ideas]
Since the build process is quite easy and we've just merged delay panning and auto adjustment for faders, I think releasing early preview versions for new features before the next release would be great. Any thoughts?

**Comment by hoffie** [comment]
Fine by me. This would be a "beta", right?
Should we settle on [what the next version will be called](https://github.com/jamulussoftware/jamulus/discussions/1302#discussioncomment-509878) first?
In my opinion, new features warrant a new minor release. This would that what you propose would become 3.8.0beta1.
